committee-code-of-conduct/charter.md

```diff
@@ -147,4 +149,12 @@ self-elected reason, they must notify the committee as well as the steering
 committee in writing. As a courtesy, such notifications should be given at least
 30 calendar days in advance of their departure.
 
+### Participation in the Kubernetes steering committee
+Participation in both of these committees represents a conflict of interest.
+If a current code of conduct committee member chooses to run for the Kubernetes steering committee, 
+they are required to announce their resignation from the code of conduct committee upon the 
+announcement of their candidacy for steering (not after the election takes place). Their resignation 
+would then take effect on the date the steering election concludes to allow notice time for a replacement 
+member to be appointed.
+
```

Comment:
Can we break this out into a separate PR? I don't think theres any debate on this part of it and we should ideally get it merged before the actual election kicks off :x